README.md

```diff
@@ -1,40 +1,46 @@
-# TRex Web Playground
+# TRex Web Playground #
 
 
 Have TRex running inside a container using your web browser.
 
 Provides a quick way to get a look-and-feel of what TRex is.
 
-![alt text](https://trex-tgn.cisco.com/trex/doc/images/trexweb.jpg)
+![trex logo](https://i.imgur.com/gx0r5PS.png)
 
 
-## Getting Started
+## Getting Started ##
 
-TRex web playground is actually  a simple skeleton meant to be enhanced.
+TRex web playground is actually a simple skeleton meant to be enhanced.
 
 It consists of a node.js server and client static HTML file.
 
 
-### Prerequisites
+### Prerequisites ###
 
 To get the server running you need:
+* [Node.js](https://nodejs.org)
+* [Docker](https://docs.docker.com/install/)
 
+### Installing ###
+
+Assuming you have node.js and Docker, do...
 
 ```
-Node.js
-Docker
+$ docker pull trexcisco/trex-dev:2.36
```

Comment:
People may not realize they need to pull the docker image, and specifically trexcisco/trex-dev:2.36 in this case.